interface/billing/billing_report.php

```diff
@@ -770,7 +770,7 @@ function MarkAsCleared(Type) {
             }
             $list = getBillsListBetween("%");
             ?>
-            <input name='bill_list' type='hidden' value="<?php echo attr($list); ?>"> <!-- new form for uploading -->
+            <input name='bill_list' type='hidden' value="<?php echo attr(implode(", ", $list)); ?>"> <!-- new form for uploading -->
```

Comment:
It looks like there should not be a space here after the comma:
```
<input name='bill_list' type='hidden' value="<?php echo attr(implode(",", $list)); ?>"> <!-- new form for uploading -->
```

Reply:
Also, should be parenthesis around it:
```
<input name='bill_list' type='hidden' value="<?php echo (empty($list)) : '' ? '('.attr(implode(", ", $list)).')'; ?>"> <!-- new form for uploading -->
```

Reply:
As an aside, I can not figure out where this is even being used in the codebase. I wonder if we can remove this line. @sunsetsystems and @sjpadgett ; am I missing something here?

Reply:
I did see that the previous formatting was set up that way.  I honestly wasn't sure if the formatting mattered, since it appears to just be the value of an element that is never used.  At least as far as I can tell, the element is always hidden on the Billing Manager page.  I was unable to find any reference of the `bill_list` field elsewhere in the code. I'm happy to change it to match the previous formatting, or to remove the element altogether.

Reply:
hi @kacyw , I done a detailed look through the code and agree the above line is not used anywhere in the codebase. So, lets remove the line.